resources/qml/WelcomePages/AddPrinterByIpContent.qml

```diff
@@ -116,14 +74,12 @@ Item
                     anchors.left: parent.left
                     height: addPrinterButton.height
                     anchors.right: addPrinterButton.left
-                    anchors.margins: 20
+                    anchors.margins: UM.Theme.getSize("default_margin").width
                     font: UM.Theme.getFont("default")
 
-                    text: ""
-
                     validator: RegExpValidator
                     {
-                        regExp: /[a-zA-Z0-9\.\-\_]*/
+                        regExp: /[0-9\.\-\_]*/
```

Comment:
Should it also accept hostnames here? And maybe IPv6 addresses? If it's only IP addresses, we can remove the dash and the underscore.

Reply:
Removed dash, underscore, added `:` and _a through f_ for ipv6 addresses.